package.json

```diff
@@ -57,7 +57,8 @@
   "license": "MIT",
   "dependencies": {
     "@improbable-eng/grpc-web": "^0.12.0",
-    "@textile/threads-client": "^0.2.6",
+    "@textile/threads-client": "^0.2.8",
```

Comment:
this is the only real change, the rest is just tests/example reformatting to use the newDB api